legion/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ endif()
 function(add_app_folder folder)
     add_executable("${folder}" "${folder}/main.cpp" "${folder}/taco-generated.cpp")
     target_include_directories("${folder}" PRIVATE "${folder}")
-    target_link_libraries("${folder}" distal-runtime-lib Legion::Legion openblas)
+    target_link_libraries("${folder}" distal-runtime-lib Legion::Legion openblas tblis)
```

Comment:
This is something that I can do as a later refactor, but we probably want to have the TBLIS path as something configurable, rather than assuming it comes from the OpenBLAS path (I'm working on something to get MKL to work, which removes the hardcoded OpenBLAS path).

Reply:
I made a CMake module file for finding TBLIS and its companion library TCI, which is probably the most proper way of doing this.